apps/fz_http/test/fz_http_web/live/user_live/show_test.exs

```diff
@@ -479,6 +479,53 @@ defmodule FzHttpWeb.UserLive.ShowTest do
     end
   end
 
+  describe "user role" do
+    setup do
+      admin_user = FzHttp.UsersFixtures.user(role: :admin)
```

Comment:
Thoughts on aliasing `FzHttp.UsersFixtures` at the top of the file?

Reply:
Sure